contents/docs/integrate/server/python.mdx

```diff
@@ -193,6 +193,65 @@ def homepage(request):
 
 > **Note:** Feature flags are persistent for users across sessions. Read more about feature flag persistence on our [dedicated page](/docs/user-guides/feature-flags#feature-flag-persistence).
 
+**Get a flag value**
+
+If you're using multivariate feature flags, you can also get not just whether the flag is enabled, but what value its enabled to.
+
+
+```python
+posthog.get_feature_flag('beta-feature', 'distinct id')
+# returns string or None
+```
+
+**Overriding server properties**
+
+Sometimes, you might want to use properties that aren't available on PostHog's servers / haven't been ingested yet / were set incorrectly earlier to evaluate feature flags. You can do so by setting properties the flag depends on with these calls.
```

Comment:
```suggestion
Sometimes, you might want to to evaluate feature flags using properties that aren't available on PostHog's servers, haven't been ingested yet, or were set incorrectly earlier. You can do so by setting properties the flag depends on with these calls.
```
I think this is a little clearer, but the original sentence could be read in two ways so will defer to you Neil